src/main/docker/scripts/start.sh

```diff
@@ -15,5 +15,5 @@
 # limitations under the License.
 #
 
-export JAVA_HOME=/usr/share/elasticsearch/jdk
+export JAVA_HOME=/usr/lib64/jvm/java-11-openjdk-11/
```

Comment:
The base image being used at the minute is `cafapi/opensuse-jre11:3`.  That image only contains a JRE and doesn't contain a JDK at all.

It presets the `JRE_HOME` environment variable but not the `JAVA_HOME` one which is meant for JDKs:
![image](https://user-images.githubusercontent.com/20627297/108252974-2ad7af80-7151-11eb-9608-df88a16c09e6.png)

Is a JDK actually required to run Elasticsearch or is a JRE sufficient?

If it is required then we need to update the base image and we shouldn't need to set this environment variable here because it will already have been set.

If a JRE is sufficient then does this mean that they want `JAVA_HOME` set to the location of the JRE?  It's wrong but it's a common misconception given Oracle's original mistake in using `JAVA_HOME` instead of `JDK_HOME`.  If this is the case then modify this line to set the environment variable using the value from `JRE_HOME` rather than a hard-coded path.

Reply:
JRE seems to be sufficient, but we need JAVA_HOME if not using the bundled jdk.